src/elements/file-selector/file-selector.ts

```diff
@@ -61,21 +67,43 @@ export class SbbFileSelectorElement extends SbbDisabledMixin(LitElement) {
   /** This will be forwarded as aria-label to the native input element. */
   @property({ attribute: 'accessibility-label' }) public accessibilityLabel: string | undefined;
 
-  /** The list of selected files. */
-  @state() private _files?: File[];
+  /** The path of the first selected file. '' no file is selected */
+  @property({ attribute: false })
+  public override set value(value: string | null) {
+    this._hiddenInput.value = value ?? '';
```

Comment:
Just for my understanding: what happens to the form value if assigning a string here (not empty)? Do the files get updated or cleared? In case it gets updated, probably not possible as there is no real file, but only a string?

Reply:
If a not-empty string is set, the native input throws an error:
`VM198:1 Uncaught InvalidStateError: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string.`

So essentially, it's an invalid input

Reply:
Ok so we don't additionally have to throw an error as the native element does itself?

Reply:
Yes, that was the idea

Reply:
ok, I resolve the conversation